localizations/fr-FR.json

```diff
@@ -0,0 +1,415 @@
+{
+    "⤡": "⤡",
+    "⊞": "⊞",
+    "×": "×",
+    "❮": "❮",
+    "❯": "❯",
+    "Loading...": "Chargement...",
+    "view": "查看",
+    "api": "api",
+    "•": "•",
+    "built with gradio": "Construit avec Gradio",
+    "Stable Diffusion checkpoint": "checkpoint Stable Diffusion",
+    "txt2img": "txt2img",
+    "img2img": "img2img",
+    "Extras": "Extras",
+    "PNG Info": "Infos PNG",
+    "History": "Historique",
+    "Checkpoint Merger": "Fusion de checkpoints",
+    "Train": "Entrainer",
+    "Settings": "Paramètres",
+    "Prompt": "Prompt",
+    "Negative prompt": "Prompt négatif",
+    "Run": "Lancer",
+    "Skip": "Passer",
+    "Interrupt": "Interrrompre",
+    "Generate": "Générer",
+    "Style 1": "Style 1",
+    "Style 2": "Style 2",
+    "Label": "Etiquette",
+    "File": "Fichier",
+    "Drop File Here": "Déposer votre fichier ici",
+    "-": "-",
+    "or": "ou",
+    "Click to Upload": "Cliquer pour uploader",
+    "Image": "Image",
+    "Check progress": "Voir l'avancement",
+    "Check progress (first)": "Voir l'avancement (1er)",
+    "Sampling Steps": "Etapes d'échantillonnage",
+    "Sampling method": "Méthode d'échantillonnage",
+    "Euler a": "Euler a",
+    "Euler": "Euler",
+    "LMS": "LMS",
+    "Heun": "Heun",
+    "DPM2": "DPM2",
+    "DPM2 a": "DPM2 a",
+    "DPM fast": "DPM fast",
+    "DPM adaptive": "DPM adaptive",
+    "LMS Karras": "LMS Karras",
+    "DPM2 Karras": "DPM2 Karras",
+    "DPM2 a Karras": "DPM2 a Karras",
+    "DDIM": "DDIM",
+    "PLMS": "PLMS",
+    "Width": "Largeur",
+    "Height": "Hauteur",
+    "Restore faces": "Restaurer les visages",
+    "Tiling": "Mode Tuile",
+    "Highres. fix": "Correction haute résolution",
+    "Firstpass width": "Largeur première passe",
+    "Firstpass height": "Hauteur seconde passe",
+    "Denoising strength": "Puissance de réduction du bruit",
+    "Batch count": "Nombre de lots",
+    "Batch size": "Taille de lots",
+    "CFG Scale": "Echelle CFG",
+    "Seed": "Graine",
+    "Extra": "Extra",
+    "Variation seed": "Variation de la graine",
```

Comment:
I think `"Variation de la graine"` is a blank translation i suggest something like `"Variation de la valeur aléatoire"`